client/guidestours/index.js

```diff
@@ -175,6 +175,7 @@ export default class GuidesTours extends Component {
 	}
 
 	quit() {
+		//TODO: should we dispatch a showGuidesTour action here instead?
```

Comment:
Good question. Probably, yes. Depends on what we consider _quitting_ to be. Plus, it's a one-liner in each case, so maybe replace with action dispatching?